assets/dev/scss/frontend/_container.scss

```diff
@@ -44,9 +44,12 @@
 	min-height: var(--min-height);
 	height: var(--height);
 	border-radius: var(--border-radius);
-	margin: var(--margin-top, var(--margin-block-start)) var(--margin-right, var(--margin-inline-end)) var(--margin-bottom, var(--margin-block-end)) var(--margin-left, var(--margin-inline-start));
-	padding-inline-start: var(--padding-left, var(--padding-inline-start));
-	padding-inline-end: var(--padding-right, var(--padding-inline-end));
+	margin-block-start: var(--margin-top, var(--margin-block-start));
+	margin-inline-end: var(--bc-margin-inline-end, var(--margin-inline-end));
+	margin-block-end: var(--margin-bottom, var(--margin-block-end));
```

Comment:
Looks like `--margin-bottom` usage should be refactored as well? 
@hein-obox

Reply:
@mykytamurzin I have updated it. It wasn't strictly necessary, but I believe it might help to avoid confusion.